requirements.txt

```diff
@@ -97,3 +95,11 @@ typing_extensions==4.11.0
 python-cas==1.6.0
 django-cas-ng==5.0.1
 asgiref==3.8.1
+
+backports.zoneinfo;python_version<"3.9"
+webencodings==0.5.1 # django-markdownify
+tinycss2==1.3.0 # django-markdownify
+bleach==6.1.0 # django-markdownify
+django-markdownify==0.9.5
+django_courseaffils==2.4.0
```

Comment:
django_courseaffils can be removed here - it looks like this project isn't using it.